app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -75,16 +89,23 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
       task.endTimestamp = DateTime.now().millisecondsSinceEpoch;
     }
 
-    await config.db.withTransaction<void>((Transaction transaction) async {
+    await datastore.db.withTransaction<void>((Transaction transaction) async {
       transaction.queueMutations(inserts: <Task>[task]);
       await transaction.commit();
     });
 
+    /// Insert data to [BigQuery] when task status is fianlized
+    /// 
+    /// [endTimestamp] greater than 0 is a good final-status flag 
```

Comment:
Nit: this comment is simply repeating the code, could be removed.

Reply:
removed.